#!/usr/bin/env python
PACKAGE = "cbot_actuators"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("mode",    str_t,    0, "Mode of UUV", "rov")
gen.add("status",    str_t,    0, "Status uf UUV", "stop")

gen.add("saturation_f",    double_t,    0, "Saturation value for common mode forward thrusters", 100.0 ,  0.001, 200.0)
gen.add("saturation_v",    double_t,    0, "Saturation value for common mode verticle thrusters", 100.0 ,  0.001, 200.0)

gen.add("thrusters_on",    int_t,    0, "Actuators swith", 0 ,  0, 1)

exit(gen.generate(PACKAGE, "Config", "Actuators"))
